sources/packages/backend/apps/test-db-seeding/src/seed-executors/data-seed.decorator.ts

```diff
@@ -0,0 +1,40 @@
+import { SetMetadata } from "@nestjs/common";
+
+// metaDataKey for the provider class.
+export const DATA_SEED = Symbol.for("DATA_SEED");
+// metaDataKey for the provider method.
+export const DATA_SEED_METHOD = Symbol.for("DATA_SEED_METHOD");
+
+// Seed priority order.
+export enum SeedPriorityOrder {
```

Comment:
I would suggest having it named as below with some comments about what is the `Priority1`.
```ts
export enum SeedPriorityOrder {
  Priority1= 1,
  Unknown = Number.MAX_SAFE_INTEGER
}
```

Reply:
ya. I liked this enum

Reply:
When I read the name unknown, felt confused we can either user Default/Least which can convey the intention better IMO.

Also, the the default priority order can be set in the implementation of decorator and not needed to be passed as `@Decorator({order:default})`

Reply:
i feel, unknown is same as not given. Its better to go with default. That conveys the difference between not given/unknown and default value.

Reply:
DEVS pls refer https://github.com/bcgov/SIMS/pull/1525#discussion_r1024453912 and let me know ur thoughts. for me both are okay. but `unknown` makes more sense